wordpress.yaml

```diff
@@ -1,7 +1,136 @@
 ---
-- name: Deploy Wordpress
+- name: Setting up wordpress
   hosts: all
   tasks:
+    - name: Update repositories 
+      become: true
+      apt:
+        #upgrade: yes
+        update_cache: yes
+        cache_valid_time: 86400 #A day
+      
+    - name: Install php dependencies
+      become: true
+      apt:
+        name:
+          - php-cli
+          - php-fpm
+          - php-mysql
+          - php-json
+          - php-opcache
+          - php-mbstring
+          - php-xml
+          - php-gd
+          - php-curl
+        state: present
+     
+    - name: Install nginx
+      become: true
+      apt:
+        name: nginx 
+        state: present
```

Comment:
after full setup could you restart your VM and check if wordpress is running?

Reply:
I tried a `vagrant halt` and `vagrant up` to check this. WP runs correctly and with no data loss after a restart.